### ISSUE TITLE
Doesn't work in Vue files despite guidelines

### ISSUE THREAD

**Issue by adi518** [vuejs/eslint-plugin-vue#455]
**Tell us about your environment**

* **ESLint Version: 3.19.0**
* **eslint-plugin-vue Version: 4.4.0**
* **Node Version: 7.10.0**

**Please show your full configuration:**

```
// https://github.com/vuejs/eslint-plugin-vue
// http://eslint.org/docs/user-guide/configuring

module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint',
    sourceType: 'module'
  },
  env: {
    browser: true,
  },
  // https://github.com/feross/standard/blob/master/RULES.md#javascript-standard-style
  extends: ['standard', 'plugin:vue/base'],
  // required to lint *.vue files
  plugins: [
    // 'html',
    'vue'
  ],
  // add your custom rules here
  'rules': {
    // allow paren-less arrow functions
    'arrow-parens': 0,
    // allow async-await
    'generator-star-spacing': 0,
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
    // #Wobi
    'space-before-function-paren': 0,
    'comma-dangle': 0,
    'no-unneeded-ternary': 0,
    'no-unused-vars': 0,
    'no-trailing-spaces': 0,
    'padded-blocks': 0,
    'space-in-parens': 0,
    'eol-last': 0,
    'no-multiple-empty-lines': 0
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
  <div>
    <!-- eslint-disable-next-line vue/require-component-is -->
    <component is="input" />
  </div>
</template>
```

**What did you expect to happen?**
For the lint to be ignored.
**What actually happened? Please include the actual, raw output from ESLint.**
Nothing.

**Comment by michalsnik** [comment]
It seems to work fine [here](https://mysticatea.github.io/vue-eslint-demo/#eJyNVstu3DgQ/JWJzkMY2KPhDRAssD+we/SFkloSY4pUmuR4FMP/niKptyZADjaGVc1+d1MfRWVrKp6LF0/9oKWnr6/mcnmp1S39wM8vQlzIaWW8qJWTpSZh6O4FELrcAj0x/QiKSVS2H6whyCl3EWK+v8AX5f5+LZQZgn8tLk/Z0FO29PK0sf/iKlaDxy+6D5b9paZGBu0vH6/mM8pOfHEtOGhyxfMHwjDOc6i8ZeHCQFw8/3UtjBWVdCRqqrRk6RWkFkJL5wT+VGsWDL5KjgG24iexXXFTn0SNs5q2R/9AxEvkI15X0fiG8Wy1YGrpPoM1laFtV89r0uRJ3OSKIDAhuXU7IAfSU18S75k3GreAVhUSnDIyo0i6H0XVITmVJ8669uQgPRizA5fD/RAyGoOlKC0yI0205PeMo17NSCO19h3b0HYLFEx10NhqW0p9AJUxcPZRVZW5Sa3qnNlhQRnnAGHx3ilPbpDVkoJe3akWCUI7oM5elos6Q+/CjX1p9YzY8jvi0noRsZWXC8uUnaIVgB+T9hlzpJtDQBBgF2vLci2Z75QTJTUWHbnr6WAwEesBgRKqV4seQ6LiXK4ck0R1S72BnGxINMogiPGAou/ltk+DCQ56tSxpDXgC0ZcrhJAITUiukkM2hTzBKeE6DHCHtCZwXhWjIp0R3MS6MDKbzLXz40A2xxf3C/qPVRnQud04dGRWD3esE5brKUeRwCz3lJYWwxF1o4XpfK/R6dYp04oSnf9GPpUI570QpXaY5m1BFdJv/B77Eayng1yq82RnYb67e0yXW/MX0V5iktZIUGux1DHxAZuk6pArg2EcrE5JODltZE9iYIv7cazlzjLqJN2IAVMmLboQW2YSVhvf835qQkrPTaD98F81W36/XVZw2jBrJFs+ded2FCY8tn40RcyWtwSTI77ByaMlh30FeH1waunlTkDVJKhpUHj3B+HOb080tMfvHpakmNO/JY9jEOHUgbPB5NrKRdMiFehQl0fv54mcnsAUwIlMquPUnO8xWhguMfnA5sTm6m6jXpg0ib+xGnU9SuuqJr/Qx0lJCw335nwvxE2UEBXOj3oLxg7fQ3k/zOVia/2By5pOIKZQvp1QLDXadvYWP4HNpjlnLI75CXygsMdH1llws8VWqDpbHviMYa++n8DYrwA/r0VO/H/qZ76Zj/+PcTcX+fW7FoRvEssTGD8Dv5n63/gY/hM/Ca9FepIQc4H3komKz19qrH8D)

And without comment it throws a warning, [here](https://mysticatea.github.io/vue-eslint-demo/#eJyNVstu5DYQ/JWBzkMYyNHwBlgskB9Ijr5QUmvENUVqm+R4tIb/PUVSD0pygBxsDKua/e6mPqrGtlQ9Vy+ehlFLT3++msvlpVX39AM/GzuM1pDxF+W+vVbKjMG/VpenLPiUJV+eivsvrmE1evyix2jZX1rqZND+8vFqPqPszFfXioMmVz1/wA3jPIfGWxYujMTV8x/XyljRSEeipUZLll5BaiW0dE7gT93MisFXySQM3cRvYrvhpj2JGmc1lUf/hYiXxqfrKhovGM9WC6YbPRawpTrcbpvnLWnyJO5yQxCYkHxzOyAHMtBQE++ZN5pKQKsGCU4ZWVAk3U+i6ZGcxhNnXXtylB6M2YHr4XEImR6epagtMiNNtOT3jKNBLUgntfY923DrVyiY5qDxpm0t9QFUxsDZr6qqzF1q1ebMjivKOAcIi/deeXKjbNYUDOpBrUgQ2gF19rJe1Rl6F24aaqsXxNY/EZfWq4htvFxZpuwUbQD8mLUvmCPdHQKCALtYW5ZbyXyvnKips+jIXU8Hg4nYDgiUUL1WDBgSpZWhjWOSqG6tC8jJjkSnDIKYDij6XpZ9Gkxw0KtlTVvAM4i+3CCERGhCco0csynkCU4J12OAe6Q1gUy/gkI0kyKdEdwUyhmZTeba+Wkkm+O7B3pC/7GqAzq3n8aezObhjnXCcjvnKBKY5QFbR7Qw2Hh1p5Xp/aDR6dYpcxM1Ov+NfCoRznshSu0wz9uKKqTf+D32K1hPB7lU59nOyvx0j5gut+UvooPEJG2RoNZirWPiAzZJ0yNXBsM4Wp2ScHLayIHEyBb341jLnWXUSboJA6ZMWnQhtswsrArf837qQkrPXaD98F91Jb/fLhs4b5gtkpJP3VmOwozH1o+miNlySTA54jucPFpy2FeA15dFtNLLnYBqSVDXofDuf4S7vD3R0B5/eFiSYkl/SR7HIMKpAxeDybWNi6ZFKtChLstQbOEodyLnJzAFcCKT6jg153uMFoZLTD6wObG5umXUK5Mm8T+sRl1fpXVTk1/o46SkhYZ7S75X4i5qiArnJ12CscP3UN4PS7nYWn/gsqYTiCmUbycUS43Kzi7xE9gVzblgccxP4BcKB3wknQWLLbZBzdnyyGcMe/X9BMZ+Bfh5rXLi/1a/8818/GeKu7nKr9+1InyTWJ7B+Bn33bR/xcfwR/yku1bpSULMFd5LJqo+/wVN0Wk0).

Does it happen in your command line, or in editor?

**Comment by michalsnik** [comment]
Also you extend `vue/base` that doesn't include `vue/require-component-is` in it's preset.

**Comment by michalsnik** [comment]
I assume it worked as there was no answer since April. Feel free to reopen if it doesn't work though :)
